### PR TITLE
Merge rc/v1.5.0 2022.12.07 2

### PR DESCRIPTION
**Pull request by iulianpascalau**
## Reasoning behind the pull request
- merge
  
## Proposed changes
- merge

## Testing procedure
- N/A

## Pre-requisites

Based on the [Contributing Guidelines](https://github.com/ElrondNetwork/elrond-go/blob/master/.github/CONTRIBUTING.md#branches-management) the PR author and the reviewers must check the following requirements are met:
- was the PR targeted to the correct branch?
- if this is a larger feature that probably needs more than one PR, is there a `feat` branch created?
- if this is a `feat` branch merging, do all satellite projects have a proper tag inside `go.mod`?

### CODE REVIEW

**Review by SebastianMarian — DISMISSED** [review]

File: factory/crypto/cryptoComponents.go [hunk 1]
```diff
@@ -235,18 +224,30 @@ func (ccf *cryptoComponentsFactory) Create() (*cryptoComponents, error) {
 		return nil, err
 	}
 
+	signatureHolderArgs := ArgsSigningHandler{
```
Comment (ssd04) [inline factory/crypto/cryptoComponents.go:227]: rename also to `signingHandlerArgs` ?

File: integrationTests/testConsensusNode.go [hunk 2]
```diff
@@ -280,13 +263,29 @@ func (tcn *TestConsensusNode) initNode(
...
+	pubKeyBytes, _ := pubKey.ToByteArray()
+	pubKeyString := coreComponents.ValidatorPubKeyConverterField.Encode(pubKeyBytes)
 	argsKeysHandler := keysManagement.ArgsKeysHandler{
 		ManagedPeersHolder: keysHolder,
 		PrivateKey:         tcn.NodeKeys.Sk,
 		Pid:                tcn.Messenger.ID(),
 	}
 	keysHandler, _ := keysManagement.NewKeysHandler(argsKeysHandler)
 
+	signatureHolderArgs := cryptoFactory.ArgsSigningHandler{
```
Comment (ssd04) [inline integrationTests/testConsensusNode.go:275]: rename here also

**Review by ssd04 — DISMISSED** [review]

File: consensus/interface.go [hunk 3]
```diff
@@ -165,11 +165,12 @@ type PeerBlacklistHandler interface {
 	IsInterfaceNil() bool
 }
 
-// SignatureHandler defines the behaviour of a component that handles signatures in consensus
-type SignatureHandler interface {
+// SigningHandler defines the behaviour of a component that handles multipart and single signatures used in consensus operations
```
Comment (AdoAdoAdo) [inline consensus/interface.go:168]: can you rename multipart to multi signature?

File: factory/crypto/cryptoComponents.go (hunk 1 above)
Comment (iulianpascalau) [reply factory/crypto/cryptoComponents.go:227]: done, thanks

File: integrationTests/testConsensusNode.go (hunk 2 above)
Comment (iulianpascalau) [reply integrationTests/testConsensusNode.go:275]: done

File: consensus/interface.go (hunk 3 above)
Comment (iulianpascalau) [reply consensus/interface.go:168]: replaced

**Review by AdoAdoAdo — APPROVED** [review]

**Review by SebastianMarian — APPROVED** [review]

**Pull request merged**